### PR TITLE
Test/KLBP-284: Document typeid incorrect.

### PR DESCRIPTION
**Pull request by iuliudascalu**

### CODE REVIEW

File: src/components/step2/Step2Content.tsx [hunk 1]
```diff
@@ -8,8 +8,13 @@ import { isJourneyRevisit } from '../../services/utils'
...
 function Step2Content(): ReactElement {
   const step2HookData = useStep2Hook()
-  const { person, onClickNext, onClickPrevious, isDisclaimerActive, journeyStatus } =
-    step2HookData
+  const {
+    person,
+    onClickNext,
+    onClickPrevious,
+    isDisclaimerActive,
+    journeyStatus,
```
Comment (ioan-iacob-netea) [inline src/components/step2/Step2Content.tsx:16]: Please sort properties.

File: src/hooks/useStep2Hook.ts [hunk 2]
```diff
@@ -105,7 +105,9 @@ const useStep2Hook = () => {
     const formParams = {
       ...data,
       currentStep: STEP_2_ID,
-      typeId: person?.identity_document_type || documentType.value,
+      typeId: '1',
+      identity_document_type:
```
Comment (ioan-iacob-netea) [inline src/hooks/useStep2Hook.ts:109]: Please use camel case for the name like the rest of the properties.

File: src/services/API/person.ts [hunk 3]
```diff
@@ -33,7 +33,15 @@ export async function createPersonWithDocuments(
...
 export function generateUploadDocuments(formData: IStep2CreatePersonParams) {
-  const { files, nom, prenom, typeId, currentStep, personTypeId } = formData
+  const {
+    files,
+    nom,
+    prenom,
+    typeId,
+    currentStep,
+    personTypeId,
+    identity_document_type,
```
Comment (ioan-iacob-netea) [inline src/services/API/person.ts:43]: Please sort the properties.

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/hooks/useStep2Hook.ts (hunk 2 above)
Comment (andreir-suciu) [reply src/hooks/useStep2Hook.ts:109]: I think we should keep this name as it is on LBP and on BE side.

**Review by andreir-suciu — APPROVED** [review]

File: src/hooks/useStep2Hook.ts (hunk 2 above)
Comment (ioan-iacob-netea) [reply src/hooks/useStep2Hook.ts:109]: Having multiple naming strategies is confusing. If we are talking about parameters being sent in API calls they can be adjusted by transformations if there is a concrete request for them to be in a certain way.

File: src/components/step2/Step2Content.tsx (hunk 1 above)
Comment (iuliudascalu) [reply src/components/step2/Step2Content.tsx:13]: Done.

File: src/services/API/person.ts (hunk 3 above)
Comment (iuliudascalu) [reply src/services/API/person.ts]: Done.

File: src/hooks/useStep2Hook.ts (hunk 2 above)
Comment (iuliudascalu) [reply src/hooks/useStep2Hook.ts]: Done.

**Review by ioan-iacob-netea — APPROVED** [review]

**Pull request merged**